taxcalc/current_law_policy.json

```diff
@@ -106,12 +112,14 @@
         "col_var": "",
         "row_var": "FLPDYR",
         "row_label": ["2013",
-		      "2014",
-		      "2015"],
+		              "2014",
+		              "2015",
+                      "2016"],
         "cpi_inflated": true,
         "col_label": "",
         "value":  [7150,
                    7250,
+                   7400,
```

Comment:
> 12 Alternative Minimum Tax Exemption for a Child Subject to the “Kiddie Tax.” For taxable years beginning in 2016, for a child to whom the § 1(g) “kiddie tax” applies, the exemption amount under §§ 55 and 59(j) for purposes of the alternative minimum tax under § 55 may not exceed the sum of (1) the child’s earned income for the taxable year, plus (2) $7,400.

https://www.irs.gov/pub/irs-irbs/irb15-44.pdf 

I'll add my priamry two sources as a comment in the main thread